docs/devGuide/contributing.md

```diff
@@ -0,0 +1,73 @@
+
+## How to contribute
+
+All contributions are welcome. You can contribute to MarkBind by:
+- Reporting bugs
+- Requesting features
+- Making changes (with a corresponding Pull Request), by:
+   - Updating documentation
+   - Fixing bugs
+   - Implementing features 
+
+### How to report a bug
+
+Report bugs in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. This helps us in undestanding your report.
+
+- Before reporting a bug, please search if someone else has has already reported it in the [existing issues](https://github.com/MarkBind/markbind/issues).
+- Is your bug from the current version or an older version of MarkBind? What OS are you on? Tell us about your environment, for example, the MarkBind version you are on.
+- What did you do? What did you expect to happen? What actually happened? 
+  - Include things such the actual MarkBind code causing the issue, the steps for reproduction, the raw output, and what you expect instead can help us in diagnosing the issue quicker
+  - You are also encouraged to submit a PR that reproduces this in `test/functional/test_site/bugs/`.
+
+Please do not submit personal support requests (eg: "How do I use X?") as bugs. Instead, feel free to [submit a blank issue](https://github.com/MarkBind/markbind/issues/new) to our issue tracker with a label of <a href="https://github.com/MarkBind/markbind/issues?q=is%3Aopen+is%3Aissue+label%3Aquestion" class="badge" style="color:white; background-color: #A41BD6;">question</a>.
+
+### How to request a feature
+
+We welcome suggestions to improve markbind as well. To do so, you can request for new features or enhancements to existing features in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guidelines below. This helps us in undestanding your request.
+
+- Before requesting a new feature, please search if someone else has has already requested for it in the [existing issues](https://github.com/MarkBind/markbind/issues).
+- Is your request related to a problem? Provide a clear and concise description of what the problem is. 
+  Eg: I have an issue when [...]
+- Describe the solution you'd like. Provide a clear and concise description of what you want to happen.
+- Describe alternatives you've considered. Let us know about other solutions you've tried or researched.
+- You can also share some additional context, anything else that you can add about the proposal.
+
+### First issues for contributors
+
+Unsure where to begin contributing to MarkBind? 
+
+We recommend to visit the [Getting Started](https://markbind.org/userGuide/gettingStarted.html) section in the User Guide and play around with Markbind first. Explore and understand the various features it provides.
```

Comment:
Suggestion:

We recommend ~to visit~ **that you start off by visiting** the [Getting Started](https://markbind.org/userGuide/gettingStarted.html) section in the User Guide and ~play around with Markbind first~ trying out MarkBind as a user.